c/prima.c

```diff
@@ -1,12 +1,47 @@
+
 /* Dedicated to the late Professor M. J. D. Powell FRS (1936--2015). */
 
 #include "prima/prima.h"
+#include <stdlib.h>
+#include <math.h>
+
+
+int prima_init_options(prima_options * opt)
```

Comment:
This function should take the number `n` of variables as an argument because some default options like `npt` and `maxfun` clearly depend on that.

Reply:
I chosed to set those to -1, and set them when n is know in prima_check_options
but the max default value was set to 100, now I'm setting it to -1 meaning 500*n